xr_sample_project/Assets/SampleViewer/Resources/Scripts/SampleSwitcher.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright 2025 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.UI;
+using UnityEngine.XR.ARFoundation;
+
+public class SampleSwitcher : MonoBehaviour
+{
+    [SerializeField] private bool isSampleViewer;
+
+    [SerializeField] private Button homeButton;
+    [SerializeField] private Button vrSceneButton;
+    [SerializeField] private Button xrTableTopSceneButton;
+
+
+    public void ChangeScene(string NextScene)
+    {
+        var currentScene = SceneManager.GetActiveScene().name;
+        SceneManager.LoadSceneAsync(NextScene);
+    }
+
+    private void Start()
+    {
+        if (!isSampleViewer)
+        {
+            homeButton.onClick.AddListener(delegate
```

Comment:
Do we need to unhook this listener? Will it ever get called twice if going back and forth between the viewer room and the tabletop sample?

Reply:
It shouldn't get called twice when reloading a scene previously in. Loading a new scene also unloads the previous scene so we should be okay.